README.md

```diff
@@ -636,7 +636,7 @@ registered in this step.
 
 ### Batch scoring
 After model training, the regression recipe provides the capability to score new data with the
-trained model.
+trained model. For more advanced ETL workflows for scoring, we recommend using [`spark_udf`](https://mlflow.org/docs/latest/python_api/mlflow.pyfunc.html#mlflow.pyfunc.spark_udf) directly instead of using this predict step.
```

Comment:
This is convenient for scoring a static dataset, e.g., the test set from a Kaggle competition. For more advanced ETL ... directly instead, because it is more flexible to handle different data scenarios.